docs/contributing.md

```diff
@@ -8,5 +8,9 @@
     * follow the style, naming and structure conventions of the rest of the project.
     * make commits atomic and easy to merge.
     * when updating documentation, please see [our guidance for documentation contributions](contributing_docs.md).
-    * apply format running `go fmt`, or the shell script `./scripts/checks.sh`
+    * when updating the `go.mod` file, please run `make tidy-all` to ensure all modules are updated. Please be aware that the lint stage could fail if this is not done.
```

Comment:
```suggestion
    * when updating the `go.mod` file, please run `make tidy-all` to ensure all modules are updated. It will run `golangci-lint` with the configuration set in the root directory of the project. Please be aware that the lint stage could fail if this is not done.
```